### ISSUE TITLE
build.py: specify commit & (no) APK for easy building/testing

### ISSUE THREAD

**Issue by obfusk** [obfusk/rbtlog#471]
Extend `scripts/build.py -v podman <appid>:<tag>`:

- `<appid>:<tag>:<commit>:<apk>`
- `commit` overrides the commit checked out (unless empty)
- `apk` overrides the APK downloaded (unless empty)
  - can be a `file://` URL for local testing
  - can be `none` (TBD) to only build, skip RB checks
- enable with e.g. `--local` or always?
- TBD: load dependencies like `apksigcopier`, `requests` on demand?

Examples:

- `scripts/build.py -v podman me.hackerchick.catima:v2.27.0::none` (local build using tag, no RB checks)
- `scripts/build.py -v podman me.hackerchick.catima:v2.27.0:7c7a05f:file:///some/dir/some.apk` (verify commit against local APK, using recipe from tag)

Feedback on the idea before I implement it is appreciated :)

**Comment by Acclorite** [comment]
Good idea! Would make RB debugging much faster.

Good luck.

**Comment by IzzySoft** [comment]
Indeed! It would also help to pre-check before a new release (e.g. when a previous one failed RB), without any modification to recipe or logs. Big :+1: from me!

**Comment by obfusk** [comment]
This is pretty much done now. Testing appreciated :)
